main.js

```diff
@@ -0,0 +1,14 @@
+let allPosts = document.querySelector("header nav ul li");
+allPosts.remove();
+
+let featuredTime = document.querySelector(".featured span");
+featuredTime.remove();
+
+let lastPost = document.querySelector("main .posts article");
+lastPost.remove();
```

Comment:
Although the tests are passing for this, you aren't actually removing the **last** article as the directions ask. You're removing the **first** article.
```suggestion
let allPosts = document.querySelectorAll("main .posts article");
let lastPost = allPosts[allPosts.length -1]
lastPost.remove();
```

As I mentioned above, `querySelector` grabs the **first** matching element, NOT the **last** matching element. In order to grab the **last** matching element out of several matching elements, you need to use `querySelectorAll` to get a node list of **all** matching elements. Node lists are very similar to arrays, so you can grab the last element of a node list in the same way that you'd access the last element of an array: `allPosts[allPosts.length -1]`.